Sources/NIOCore/NIOLoopBound.swift

```diff
@@ -107,6 +107,26 @@ public final class NIOLoopBoundBox<Value>: @unchecked Sendable {
         return .init(_value: nil, uncheckedEventLoop: eventLoop)
     }
 
+    /// Initialise a ``NIOLoopBoundBox`` with a ``Sendable`` value, validly callable off `eventLoop`.
+    ///
+    /// Contrary to ``init(_:eventLoop:)``, this method can be called off `eventLoop` because we know that `value` is ``Sendable``.
+    /// So we don't need to pretect `value` itself, we just need to protect the ``NIOLoopBoundBox`` against mutations which we do because the ``value``
```

Comment:
```suggestion
    /// So we don't need to protect `value` itself, we just need to protect the ``NIOLoopBoundBox`` against mutations which we do because the ``value``
```

Reply:
thanks!